docs/api/config-manager.md

```diff
@@ -0,0 +1,249 @@
+# ConfigManager
+
+The `ConfigManager` can and should be used to lookup device configuration files, meters, notifications, etc. without having to parse the config files yourself.
+You can (and should) access it through the `configManager` property of the driver instance. If that is not possible, import it from `@zwave-js/config`.
+
+## Constructor
+
+The constructor takes no arguments.
+
+```ts
+new () => ConfigManager
```

Comment:
Is this correct?

Reply:
Kinda. It accepts a log container, but you really don't want to do that outside the driver.

Reply:
Oh I just saw the suggestion. The method here is the **constructor type**, but IMO I'd probably just remove the signature.